views/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="stylesheet.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <meta charset ="utf-8">
+  <title>Benchmark Calculator</title>
+</head>
+<body>
+
+  <div class="calculator-container">
```

Comment:
Consider renaming this to describe the content, not the markup - `calculator` instead of `calculator-container`.